include/elliptics/interface.h

```diff
@@ -244,7 +244,7 @@ struct dnet_backend_callbacks {
 			struct dnet_iterator_request *ireq, struct dnet_iterator_range *irange);
 
 	int			(* defrag_status)(void *priv);
-	int			(* defrag_start)(void *priv);
+	int			(* defrag_start)(void *priv, enum dnet_backend_defrag_level level);
```

Comment:
You can break API here, since there are no other server (i.e. users of dnet_backend_callback structure) out of the elliptics tree, but you can not do that with eblob, since there will always be elliptics servers compiled with older eblob library which expects old eblob_defrag_start() function